patches/gcc/6.3.0/950-bionic-android.patch

```diff
@@ -0,0 +1,17618 @@
+Based on https://github.com/crystax/android-toolchain-gcc-6/commits/master
```

Comment:
There were some Makefile.in changes to call the target libgnustl_shared instead of libstdc++. I've removed these Makefile changes from the patch set.